packages/main/cypress/specs/MultiInput.cy.tsx

```diff
@@ -69,4 +88,1235 @@ describe("MultiInput Web Component", () => {
 			.find(`#${valueHelpId}`)
 			.should("have.text", MULTIINPUT_VALUE_HELP.defaultText);
 	});
+	it("tests expanding of tokenizer", () => {
+		cy.mount(
+			<MultiInput id="basic-overflow">
+				<Token slot="tokens" text="Amet"></Token>
+				<Token slot="tokens" text="Incididunt"></Token>
+				<Token slot="tokens" text="laboris"></Token>
+			</MultiInput>
+		);
+
+		cy.get("ui5-multi-input")
+			.as("multiInput");
+
+		cy.get("@multiInput")
+			.shadow()
+			.find("input")
+			.as("input");
+
+		cy.get("@multiInput")
+			.shadow()
+			.find("ui5-tokenizer")
+			.as("tokenizer");
+
+		cy.get("@input")
+			.click();
+
+		cy.get("@input")
+			.realPress("Tab");
+
+		cy.get("@tokenizer")
+			.should("not.have.a.property", "expanded");
+	});
+
+	it("tests opening of tokenizer Popover", () => {
+		cy.mount(
+			<MultiInput id="basic-overflow">
+				<Token slot="tokens" text="Amet"></Token>
+				<Token slot="tokens" text="Incididunt"></Token>
+				<Token slot="tokens" text="laboris"></Token>
+			</MultiInput>
+		);
+
+		cy.get("ui5-multi-input")
+			.as("multiInput");
+
+		cy.get("@multiInput")
+			.shadow()
+			.find("ui5-tokenizer")
+			.as("tokenizer");
+
+		cy.get("@tokenizer")
+			.shadow()
+			.find(".ui5-tokenizer-more-text")
+			.as("nMoreLabel");
+
+		cy.get("@nMoreLabel")
+			.click();
+
+		cy.get("@tokenizer")
+			.shadow()
+			.find("ui5-responsive-popover")
+			.as("respPopover");
+
+		cy.get("@respPopover")
+			.should("have.attr", "open");
```

Comment:
there is custom command that validates whether the responsive popover is opened. please consider using it. it will make test more stable